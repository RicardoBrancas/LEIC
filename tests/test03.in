A BBS 6
A KBH 21
A EQX 37
A GBK 2
A JDG 18
A JBT 11
A GQF 18
A SRZ 13
A WRF 13
A OXR 11
G KBH GBK
R SRZ BBS
S KBH GBK
R GQF BBS
R WRF BBS
C GBK
S WRF GQF
R GBK KBH
R KBH GBK
F BBS EQX
C DRZ
A LFK 32
F JDG GQF
A PPD 2
S SRZ OXR
G KBH OXR
Q
L 2
Q
Q
F LFK KBH
G PPD JDG
N GQF BBS
G SRZ WRF
R JDG GQF
R JBT GBK
C PPD
P
R GQF JBT
I PPD -46
G OXR GQF
G SRZ JDG
Q
N BBS SRZ
A IHG 35
G KBH LFK
Q
I IHG -33
S GQF SRZ
G IHG JDG
S WRF KBH
P
A YFX 39
I BBS 5
G GQF IHG
G BBS JBT
F EQX OXR
G GQF KBH
G BBS JBT
G BBS JBT
F EQX JDG
G GQF SRZ
G BBS JBT
G BBS JBT
F YFX EQX
G GQF YFX
P
Q
V
L 2
X