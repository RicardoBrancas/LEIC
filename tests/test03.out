Voos entre cidades LAX:OPO:1:1
*Impossivel remover voo RT LAX OPO
Voos entre cidades LAX:OPO:0:0
Voos entre cidades LAX:OPO:0:0
LAX:10:0:0
LIS:20:1:1
OPO:30:1:1
2:3
